prism_compile.c

```diff
@@ -4446,6 +4450,10 @@ pm_compile_node(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *const ret,
             }
         }
 
+        if (keyword_rest) {
```

Comment:
Such a nitpick, but can we call this something like `block_param_keyword_rest` to disambiguate it from the below?